api/envoy/api/v2/discovery.proto

```diff
@@ -55,6 +55,14 @@ message DiscoveryRequest {
   // internal exception related to the failure. It is only intended for consumption during manual
   // debugging, the string provided is not guaranteed to be stable across Envoy versions.
   google.rpc.Status error_detail = 6;
+
+  // [#not-implemented-hide:]
```

Comment:
We can probably unhide this, as there are no Envoy implementation implications.

Reply:
Done.